src/components/button/components/button-group.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import modifiers from '../../../modifiers';
+import Element from '../../element';
+
+const ButtonGroup = ({
+  children,
+  className,
+  hasAddons,
+  position,
+  renderAs,
+  ...allProps
+}) => (
+  <Element
+    renderAs="div"
```

Comment:
Can you use the `renderAs` prop instead of a `div`? And put the default value of renderAs to `div`

Reply:
What values are allowed in the `renderAs` props? React components and? Thanks!

Reply:
`strings` and `functions` (for react components)

Reply:
oneOf? or just strings?

Reply:
```
renderAs: PropTypes.oneOfType([
    PropTypes.string,
    PropTypes.func,
  ]),
```
Like this. =)

Reply:
OK! I thought you would want to limit the string to several HTML tags...